include/NovelRT/Timing/StepTimer.h

```diff
@@ -43,13 +41,13 @@ namespace NovelRT::Timing {
       return _totalTicks;
     }
 
-    inline double getElapsedSeconds() const {
+    inline Timestamp getElapsedSeconds() const {
```

Comment:
This (and the below) should probably be called `getElapsedTime()` and `getTotalTime()` now.

Reply:
Same with `getTargetElapsedSeconds` to `getTargetElapsedTime`